src/mappings/encoding.ts

```diff
@@ -180,3 +193,37 @@ export function encodeVaultId(vaultId: VaultIdV1021000) {
         collateral
     )}`;
 }
+
+/**
+ * Decodes a given ss58 encoded address, then returns the string representation of the decoded array.
+ * @param ss58Address Address in ss58 encoded format
+ * @param ss58Prefix (Optional) ss58 prefix to use in decoding, defaults to Interlay/Kintsugi prefix value
+ * @returns The decoded address as string.
+ */
+export function ss58AddressToString(ss58Address: string, ss58Prefix?: number): string {
+    const prefix = ss58Prefix !== undefined ? ss58Prefix : address.interlay.prefix;
```

Comment:
Is the ss58 prefix same for both kintsugi and interlay?

Reply:
No, the prefix is 2092 for Kintsugi and 2032 for Interlay

Reply:
What might be confusing here is that the naming of `address.interlay.prefix` is a bit unfortunate (the "interlay" part).
`address.interlay` is initialized with a codec helper from squid, and will represent kintsugi or interlay.

That name is an artifact from much earlier work. I might as well quickly refactor that as it has caused confusion for greg in another spot, too.

Reply:
Refactored the name in commit c9a3943

`address.interlay` is now `address.parachain`